packages/fxa-settings/src/components/DataBlock/index.tsx

```diff
@@ -47,11 +51,13 @@ export const DataBlock = ({
           valueIsArray ? 'py-4' : 'py-5'
         }`}
         data-testid={dataTestId}
+        onCopy={onCopy}
```

Comment:
Not a blocker, but I know @LZoog would point this out: you can de-dupe by spreading the props (`{...{ onCopy }}` instead of `onCopy={onCopy}`)

Reply:
Definitely better style. Resolved.